sdk/monitor/azure-monitor-opentelemetry-autoconfigure/pom.xml

```diff
@@ -14,7 +14,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-monitor-opentelemetry-autoconfigure</artifactId>
-  <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-monitor-opentelemetry-autoconfigure;current} -->
+  <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-monitor-opentelemetry-autoconfigure;current} -->
```

Comment:
Is this expected to point to a nonexistent "future" version?

Reply:
Yes, think of the `pom.xml` as similar to the `csproj` file.
This is specifying the next version for the autoconfigure project.

All the other pom files in this PR are referring to the 1.2.0 version that was just released.